tech_grants_pipeline_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export PYTHONPATH='.'
+# Creating training data
+python nutrition_labels/create_training_data.py --config_path configs/training_data/2021.03.08.test.ini &
+# Creating evaluation data
+python nutrition_labels/create_training_data.py --config_path configs/training_data/2021.03.29.test.epmc.ini &
+python nutrition_labels/create_training_data.py --config_path configs/training_data/2021.03.29.test.rf.ini &
+# Training models
+python nutrition_labels/grant_tagger.py --config_path configs/train_model/2021.04.02.test.ini &
+# Predicting tech grants
+python nutrition_labels/predict.py --config_path configs/predict/2021.04.02.test.ini &
+# Evaluating tech grants
+python nutrition_labels/evaluate.py --config_path configs/evaluation/2021.04.02.test.ini
```

Comment:
I noticed that you use multiple config for the same version, I think that makes it harder to read all the parameters of one pipeline experiment. You also have to remember the names of all those configs, why not have one config per version that contains information for all the pipeline? So this file would essentially use one config.

You could have a different config for another pipeline or not. Btw if you have one config per version then creating a test config is easier, you just replace with the test data. You could even accept the config as an argument to the script so that you can reuse the same pipeline script. The config does not need to set in stone. You can remove some duplication like that.

`./run_pipeline.sh configs/2021.04.02.ini`

Reply:
that makes sense! I'll have a go at a mega-config and hopefully I haven't named any variables with the same name in the individual configs